README.md

```diff
@@ -1,3 +1,7 @@
+# CI
```

Comment:
Landing page is for people that see the project for the first time. CI results are irrelevant and confusing at that point. Please add this section to CONTRIBUTING.md. It should include explanation what is flakefinder good for.